aces/Loris
# [Timepoint List] updated module help text

Updated the Timepoint List module help text

#### Testing instructions

1. Edit copy as necessary
2. see screenshot for rendering in LORIS

#### Links to related tickets

* https://github.com/aces/Loris/issues/5576#issuecomment-556905146

![Screenshot 2019-11-24 18 48 05](https://user-images.githubusercontent.com/37813606/69503739-f859eb00-0eea-11ea-85a9-a621be910fde.png)
